chopper_generator/lib/src/generator.dart

```diff
@@ -355,17 +355,17 @@ class ChopperGenerator extends GeneratorForAnnotation<chopper.ChopperApi> {
     return refer("Request").newInstance(params, namedParams);
   }
 
-  Expression _genereteMap(Map<ParameterElement, ConstantReader> queries) {
+  Expression _generateMap(Map<ParameterElement, ConstantReader> queries) {
     final map = {};
     queries.forEach((p, ConstantReader r) {
       final name = r.peek("name")?.stringValue ?? p.displayName;
-      map[literal(name)] = refer(p.displayName);
+      map[literal(name)] = refer("${p.displayName}.toString").call([]);
```

Comment:
I think u could simplify a little, what do thing about it? 

```diff
- map[literal(name)] = refer("${p.displayName}.toString").call([]);
+ map[literal(name)] = refer("${p.displayName}").call([]);

Reply:
I don't know what code will be generate, but something like `map['key'] = '$value'` could be cleaner

Reply:
@hjJunior The type of  p.displayName is String, so ${p.displayName} is an equation of p.displayName.
But the ${p.displayName}.toString is a method, so that can generate the code like "1.toString()".